components/vc/shared/wheelSpeed/wheelSpeed.c

```diff
@@ -37,40 +38,46 @@ typedef struct
 
 static wheelSpeed_S wheels;
 
+/******************************************************************************
+ *                     P R I V A T E  F U N C T I O N S
+ ******************************************************************************/
+
+static void calculateAxleSpeed(void)
+{
+    wheels.rpm_axle[AXLE_FRONT] = (uint16_t)(wheels.rpm_tire[WHEEL_FL] + wheels.rpm_tire[WHEEL_FR]) / 2;
+    wheels.rpm_axle[AXLE_REAR] = (uint16_t)(wheels.rpm_tire[WHEEL_RL] + wheels.rpm_tire[WHEEL_RR]) / 2;
```

Comment:
yikes, big oversimplification... what will  this be used for?

While I can _maybe_ see a world where a rear axle speed could be used for something (and that's a big maybe), IMO front axle speed makes no sense at all... There is no axle. There is nothing constraining the front wheels to one another

Reply:
Im going to use these to bringup a basic traction control PID that will be feature flagged out. I know front axle speed doesnt really make sense here especially in cornering. My starting point for traction control being worthwhile is in accel, where you can (while smoking a pipe) assume that the front axle is welded

Reply:
yeah fair enough but you might be doing throwaway work on your pid tuning and other calibration/design on the torque limiting side

given that you can't go out and test for a while anyway, feels like the right move is to just try to work towards the correct solution